_commands/scripting/yes.md

```diff
@@ -13,12 +13,24 @@ y
 y
 ~~~
 
-<!--more-->
```

Comment:
You'll still need a more break somewhere -- check out the other examples and the /reference page to understand what this does